unit-test

```diff
@@ -14,7 +14,7 @@ build=$(mktemp -d -p $here)
 cd $build
 
 # do clang-tidy during compilation
-../cmake-init.sh -DBUILD_WITH_CLANG_TIDY=ON
+../cmake-init.sh
```

Comment:
should we really remove the ability to run clang tidy, or just disable it by default for CI until we can fix?

Reply:
I just removed the clang tidy verification that happens during the source code building, I added it initially and it's kind of duplication of what the aktualizr clang tidy functionality does. So there is still clang tidy the one that aktualizr provides.